db/model/model.js

```diff
@@ -47,4 +47,25 @@ exports.selectArticleById = (article_id) => {
     )
 
   }
-    
\ No newline at end of file
+    
+
+
+
+
+
+
+
+
+
+
+
+  exports.removeComment = (comment_id) => {
+    return db.query('SELECT * FROM comments WHERE comments.comment_id = $1;', [comment_id])
```

Comment:
To save two queries, a SELECT and a DELETE, you could have a look at what the response object for the delete query below returns - is there anything that changes about the query depending on whether the comment exists or not?
(This isn't mandatory though just a potential improvement)